src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java

```diff
@@ -76,101 +80,111 @@ public enum PropertyType {
 		 */
 		HAVE
 	}
-	
-	@SuppressWarnings("null")
+
 	private Expression<? extends T> expr;
-	
+
 	/**
+	 * Registers a property condition. Use [!INSERT HERE] to set optional patterns for the negated part of the syntax.
```

Comment:
`PATTERN` is easier to understand than `INSERT HERE`
```suggestion
	 * Registers a property condition. Use [!PATTERN] to set optional patterns for the negated part of the syntax.
	 * Example <code>[!no|a] [direct] line of sight</code> this means if <code>no</code> part was used it will mark the pattern as negated.
```